### PR TITLE
Feature lpdev 244 search for merchant

### PR DESCRIPTION
**Pull request by olegtrofimenko**

### CODE REVIEW

File: src/AzureRepositories/AzureRepoBinder.cs [hunk 1]
```diff
@@ -410,10 +410,10 @@ public static void MapEntities()
                 AzureTableStorage<InternalTransferEntity>.Create(dbSettings.ConnectionString(x => x.InternalTransactionsConnectionString),
                     "InternalTransferInfo", log)));
 
-            if (!string.IsNullOrWhiteSpace(pdfGeneratorConnectionString?.CurrentValue))
-            {
-                container.RegisterInstance<IPdfGeneratorRepository>(new PdfGeneratorRepository(AzureBlobStorage.Create(pdfGeneratorConnectionString), log));
-            }
+            //if (!string.IsNullOrWhiteSpace(pdfGeneratorConnectionString?.CurrentValue))
```
Comment (tarurar) [inline src/AzureRepositories/AzureRepoBinder.cs]: Please remove comments

File: src/Core/Settings/IDbSettings.cs [hunk 2]
```diff
@@ -267,11 +267,11 @@ public class BcnReportsSettings
         public string ApiUrl { get; set; }
     }
 
-    public class PdfGeneratorSettings
-    {
-        public string ServiceUrl { get; set; }
-        public string ConnectionString { get; set; }
-    }
+    //public class PdfGeneratorSettings
```
Comment (tarurar) [inline src/Core/Settings/IDbSettings.cs]: Please remove comments

File: src/Lykke.Service.PayBackoffice/Binders/AzureBinder.cs [hunk 3]
```diff
@@ -130,7 +130,7 @@ public ContainerBuilder Bind(IConfigurationRoot configuration, ContainerBuilder
             var cacheManager = new MemoryCacheManager();
             ioc.RegisterInstance<ICacheManager>(cacheManager);
             BindMicroservices(ioc, settings.CurrentValue, Log);
-            ioc.BindAzureRepositories(settings.Nested(x => x.BackOffice.Db), settings.Nested(x => x.SmsNotifications), settings.ConnectionString(x => x.BackOffice.PdfGenerator.ConnectionString), settings.CurrentValue.BackOffice.DefaultWithdrawalLimit, cacheManager, Log);
+            ioc.BindAzureRepositories(settings.Nested(x => x.BackOffice.Db), settings.Nested(x => x.SmsNotifications), /*settings.ConnectionString(x => x.BackOffice.PdfGenerator.ConnectionString),*/ settings.CurrentValue.BackOffice.DefaultWithdrawalLimit, cacheManager, Log);
```
Comment (tarurar) [inline src/Lykke.Service.PayBackoffice/Binders/AzureBinder.cs]: Please remove comments

**Review by tarurar — APPROVED** [review]

File: src/AzureRepositories/AzureRepoBinder.cs (hunk 1 above)
Comment (vitaliidasaev) [reply src/AzureRepositories/AzureRepoBinder.cs]: Also please remove such Settings sections

File: src/Core/Settings/IDbSettings.cs (hunk 2 above)
Comment (vitaliidasaev) [inline src/Core/Settings/IDbSettings.cs]: Also please remove such Settings sections

**Pull request merged**